ProjectName/src/screens/PaymentDetails/PaymentDetailsForm/components/customButton/CustomButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React, {FC} from 'react';
+import {View, Text, Pressable} from 'react-native';
+
+import {styles} from './custom-button.styles';
+import {CustomButtonType} from './custom-button.types';
+const CustomButton: FC<CustomButtonType> = ({title, onPressFunction}) => {
```

Comment:
Common approach for naming `onPress`. I would suggest remove `Function` affix. It's a bit redundand in my opinion.